### PR TITLE
DIV-4446 - fixed incorrect mappings for ReceivedAOSfromResp and Date

### PR DESCRIPTION
**Pull request by hosainnet**
# Description

* Changed ReceivedAOSfromRespDate to be a date field
* Changed ReceivedAOSfromResp answer to be "Yes" instead of "YES"
Fixes #DIV-4446 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/integrationTest/resources/fixtures/divorcetoccdmapping/ccd/aos.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "RespPhoneNumber" : "12344555",
     "RespStatementOfTruth" : "YES",
     "RespConsentToEmail" : "NO",
-    "ReceivedAOSfromResp" : "Yes",
-    "ReceivedAOSfromRespDate" : "2001-02-02",
+    "ReceivedAOSfromResp" : "YES",
+    "ReceivedAOSfromRespDate" : "2001-02-03",
```
Comment (hosainnet) [inline src/integrationTest/resources/fixtures/divorcetoccdmapping/ccd/aos.json]: files in CCD should always have a "YES" and "yyyy-mm-dd" format

File: src/integrationTest/resources/fixtures/divorcetoccdmapping/divorce/aos.json [hunk 2]
```diff
@@ -284,7 +284,7 @@
     "respCostsReason" : "Some Reason",
     "respEmailAddress" : "est@somemail.com",
     "receivedAosFromResp" : "Yes",
-    "receivedAosFromRespDate" : "2001-02-02",
+    "receivedAosFromRespDate" : "2001-02-03T00:00:00.000Z",
```
Comment (hosainnet) [inline src/integrationTest/resources/fixtures/divorcetoccdmapping/divorce/aos.json]: files in `divorce` folders should always have a "Yes" and a full date/time format

File: src/integrationTest/resources/fixtures/divorcetoccdmapping/ccd/aos.json (hunk 1 above)
Comment (qzhou-hmcts) [reply src/integrationTest/resources/fixtures/divorcetoccdmapping/ccd/aos.json]: Really minor "2001-02-13"

Comment (hosainnet) [reply src/integrationTest/resources/fixtures/divorcetoccdmapping/ccd/aos.json]: Why 13?

**Comment by hosainnet** [comment]
@qzhou-hmcts I'll be testing COS against this

File: src/test/resources/fixtures/divorcetoccdmapping/divorce/aos.json [hunk 3]
```diff
@@ -283,7 +283,7 @@
     "respCostsAmount" : "1232321",
     "respCostsReason" : "Some Reason",
     "respEmailAddress" : "est@somemail.com",
-    "receivedAosFromRespDate" : "2001-02-02",
+    "receivedAosFromRespDate" : "2001-02-03T00:00:00.000Z",
```
Comment (hosainnet) [inline src/test/resources/fixtures/divorcetoccdmapping/divorce/aos.json]: Just making this different from another date value

**Comment by nsingamsetty** [comment]
Merging this to fix the mapping issue

**Pull request merged**